kong/acl_service_test.go

```diff
@@ -137,6 +137,50 @@ func TestACLGroupGet(T *testing.T) {
 	require.NoError(client.Consumers.Delete(defaultCtx, consumer.ID))
 }
 
+func TestACLGroupGetByID(T *testing.T) {
+	RunWhenDBMode(T, "postgres")
+
+	assert := assert.New(T)
+	require := require.New(T)
+
+	client, err := NewTestClient(nil, nil)
+	require.NoError(err)
+	assert.NotNil(client)
```

Comment:
```suggestion
	require.NotNil(client)
```
Using `require` here would be better to avoid potential panics. Let's make this change in other cred tests too (the ones you are adding in this PR).

Reply:
This should be changed in all places.